Form-Controls/index.html

```diff
@@ -13,15 +13,67 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
+        <section>
+          <fieldset>
+  <h2>Customer information</h2>
+     <p>
+    <label for="name">
+      <span>Full Name: </span>
+    </label>
+    <input type="text" id="name" name="username" required pattern="[A-Za-z\s]{2,}"
+              title="Please enter a valid name (at least 2 letters)."/>
```

Comment:
It's really good  way to  handle validation with pattern keyword. and  also provide   reasonable response to end user.
Well done.